fdbserver/workloads/EncryptionOps.actor.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * EncryptionOps.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbclient/DatabaseContext.h"
+#include "fdbclient/NativeAPI.actor.h"
+#include "flow/IRandom.h"
+#include "flow/StreamCipher.h"
```

Comment:
As this file includes StreamCipher, it should be compiled under the `ENCRYPTION_ENABLED` condition

Reply:
Good catch! Thanks for pointing it out.